### ISSUE TITLE
[FEATURE] Make it possible to pass more inputs to the model

### ISSUE THREAD

**Issue by RedTachyon** [GRAAL-Research/poutyne#44]
I came across this while trying to fine tune BERT using poutyne. One of the features of BERT is an input mask, so the input to the model consists of three parts (input, mask, label), which doesn't seem to be supported by poutyne.

I'm thinking enabling keyword arguments beyond (x, y) in model.fit(...) could make this work?

**Comment by freud14** [comment]
Hi, using the current master branch, x and y can both be tuple and will be unpacked when passing to the network. Thus, if `x = (a, b)`, then `y_pred = net(a, b)`. So with BERT, you could have `x, y = (input, mask), (label, mask)`. In this case, the loss function has to unpack the label and the mask before them. Is this clear?

**Comment by freud14** [comment]
@RedTachyon Do you have other questions?
